dali/test/python/test_pipeline.py

```diff
@@ -2155,3 +2155,52 @@ def create_test_package(output_dtype=None, output_ndim=None, cast_labels=False):
         create_test_package(output_dtype=int)
     with assert_raises(ValueError, glob="*types.NO_TYPE*"):
         create_test_package(output_dtype=types.NO_TYPE)
+
```

Comment:
Nitpick: How about adding a simple test (maybe in checkpointing tests?) to make sure that the checkpointing is not supported with foreign ops?

Reply:
Usual answer: I'd gladly add them, but that's waiting another.... day? for CI and I'm already on top of way too many changes with further development. I can file a separate PR just for that, though.

Reply:
Sure, that's why I marked this as nitpick